src/Wumpus.Net.Core/Entities/Permissions/ChannelPermissions.cs

```diff
@@ -2,37 +2,58 @@
 
 namespace Wumpus.Entities
 {
-    /// <summary> xxx </summary>
+    /// <summary> https://discordapp.com/developers/docs/topics/permissions#permissions </summary>
     [Flags]
     public enum ChannelPermissions : ulong
     {
-        None = 0,
-        CreateInstantInvite = 1 << 0,
-        //KickMembers = 1 << 1,
-        //BanMembers = 1 << 2,
-        //Administrator = 1 << 3,
-        ManageChannel = 1 << 4,
-        //ManageGuild = 1 << 5,
-        AddReactions = 1 << 6,
-        ReadMessages = 1 << 10,
-        SendMessages = 1 << 11,
-        SendTTSMessages = 1 << 12,
-        ManageMessages = 1 << 13,
-        EmbedLinks = 1 << 14,
-        AttachFiles = 1 << 15,
-        ReadMessageHistory = 1 << 16,
-        MentionEveryone = 1 << 17,
-        UseExternalEmojis = 1 << 18,
-        Connect = 1 << 20,
-        Speak = 1 << 21,
-        MuteMembers = 1 << 22,
-        DeafenMembers = 1 << 23,
-        MoveMembers = 1 << 24,
-        UseVAD = 1 << 25,
-        //ChangeNickname = 1 << 26,
-        //ManageNicknames = 1 << 27,
-        ManagePermissions = 1 << 28,
-        ManageWebhooks = 1 << 29,
-        //ManageEmojis = 1 << 30
+        None                = 0,
+
+        // General
+        /// <summary> Allow creation of instant <see cref="Invite"/>s. </summary>
+        CreateInstantInvite = 0x00_00_00_01,
```

Comment:
This format is inconsistent with some of the other flags enums. I don't see any benefit to this over the old bit positions but everything should be consistent.

Reply:
The only benefit I considered was being able to eyeball the hexadecimal value at a glance, as Discord's docs list out the permission values in that format. If consistency is still desired in this case, I'll swap it back to the bit shifts.

Reply:
Ah, if it's that way in the docs, we could use the same format. In that case I'd prob get rid of the separators.